### DISCUSSION TITLE
Issues with spurious elements on the page.

### DISCUSSION THREAD

**Discussion by onionjake** [jfyne/live#33] [General]
I'm seeing issues when removing elements from the page. Are there any limitations to the DOM patching? Are there any limitations to what can be used inside the go templates?

I'm trying to narrow down a case that reproduces the issue so I can file a full-on issue.

**Comment by onionjake** [comment]
Here is one example.

![image](https://user-images.githubusercontent.com/113088/140954631-2c0111a3-5fdc-45a5-9563-13eafd41ab7e.png)

For some reason a "wishlist-row" got inserted inside of another row (they should never do that).  Perhaps because it is doing prefix matching and `_l0010121` is a prefix of `_l00101210`?

It seems like it is once more than 10 of these `wishlist-rows` are created that the issues start... something to do with wrapping around at `10`?

**Reply by jfyne** [reply]
I must have just introduced this bug. Sounds like more than 10 might be the cause.

I'll try to replicate today and fix. Thanks for reporting!

**Reply by jfyne** [reply]
@onionjake try this https://github.com/jfyne/live/releases/tag/v0.12.1

In my mind I can see what the problem is, but figuring out a test case for it was too hard for me this early in the morning. I'm pretty sure that release will fix it though.
